gammapy/maps/region/geom.py

```diff
@@ -853,11 +852,7 @@ def plot_region(self, ax=None, kwargs_point=None, path_effect=None, **kwargs):
             kwargs.setdefault("facecolor", "None")
             kwargs.setdefault("edgecolor", "tab:blue")
             kwargs_point.setdefault("marker", "*")
-
-            for key, value in kwargs.items():
-                key_point = ARTIST_TO_LINE_PROPERTIES.get(key, None)
-                if key_point:
-                    kwargs_point[key_point] = value
```

Comment:
this is suppose to give the same properties to points using kwargs is kwargs_point is not specified, it should not be remove

Reply:
I found that this makes it impossible to set the colour of the points through `kwargs_point`.
Removing these lines makes no difference to the default plot, and the choices can still be passed cleanly